modules/ROOT/pages/batch-processing.adoc

```diff
@@ -616,12 +635,17 @@ image::query4.png[]
 [discrete.view]
 === Studio Visual Editor
 
-image::example-filter3.png[example_filter3]
+image::mruntime-batch-example-6.png[Insert Lead view]
+
+** The first item instructs the batch step not to process any record enriched with an `exists` record variable that equal to true.
```

Comment:
I would revert the sentence to match the expression of this example, since it is checking if `'exists' == false`:

The first item instructs the batch step to process only those records enriched with an `exists` record variable that equals to `false`.